.github/workflows/helm-release.yaml

```diff
@@ -1,12 +1,14 @@
-name: Release Helm Charts
+name: Helm Release
 
 on:
   push:
+    branches:
+      - main
     paths:
       - "helm/**"
 
 jobs:
-  release:
+  helm-release:
```

Comment:
Consider using a more descriptive name for the job, such as 'release-helm-charts', instead of 'helm-release'. This can improve readability and understanding of the workflow. [medium]